### ISSUE TITLE
Offerring (free) 1st-level support for a game using RakNet.

### ISSUE THREAD

**Issue by Luke1410** [OculusVR/RakNet#119]
We are currently looking for a game which has already been released, is utilizing RakNet, and would be interested in a cooperation with us (in principle we'd provide 1st-level support from the network engine side while we'd like to then use the product as a test bed for our SLikeNet integration). If you are interested in such a cooperation, please let us know.

**Comment by BenHandsley** [comment]
I'm looking to re-start some code that a developer released to the public. The code is for a multiplayer modification towards a single player game. The code has some issues, such as injecting an DLL into the game, but not doing anything else, instead of crashing on loading the game's profile.

Please let me know if this is what you can help with, thanks.

**Comment by Luke1410** [comment]
Hi Ben, thanks for reaching out to us.

Can you give us some more details about the project/code?
Is the code you are referering to using RakNet currently?
Is the project in active use by a community?

If so, we'd certainly consider handling this under the described offer.
Please understand that otherwise we can't cover this as part of this offer unfortunately.
However, even in this case we'll gladly provide you with any support from our side you might require to get things started. Best way would be to ask questions then in our Forum (https://www.slikesoft.com/forum/ ) and we'll lend a hand to the best of our abilities.

Feel free to contact me directly by mail at luke1410@slikesoft.com .

Regards,
Stefan

**Comment by BenHandsley** [comment]
Hello Luke,

Thanks for the reply, I have sent you an direct e-mail back towards the email that you sent me in your reply.

Hope to see an response back from yourself as soon as possible :)

Thanks,
Ben.

**Comment by dev-oren** [comment]
@BenHandsley Hello, out of simple curiosity, which Singleplayer game is it?

**Comment by BenHandsley** [comment]
@orenii The singleplayer game is Euro Truck Simulator 2.

**Comment by dschwartz783** [comment]
I am currently using it in an ongoing project to write a Minecraft PE server, primarily in the Swift programming language. However, I'm using C bindings, and then importing them into Swift, in order to use RakNet. I would be interested in further developments. It isn't working completely yet, but you can see the project here: https://github.com/DDSSwiftTech/SwiftMine

I fully intend to make extensive use of RakNet in the future. For now, its primary purpose is to interact with Minecraft.

**Comment by Luke1410** [comment]
@dschwartz783 , feel free to use our Forum to ask for any support (be it with RakNet or with migrating to SLikeNet) (https://www.slikesoft.com/forum/).

We'll pick up any support requests/questions directly there and will also help you, if you have any issues with the SLikeNet migration procedure.

**Comment by Subtixx** [comment]
Your forum doesn't seem very active. Click the first category and saw posts from a spambot.. (Which was posted more than half a month ago.)
![](https://i.imgur.com/kVMfpJo.png)

**Comment by Luke1410** [comment]
Yep, unfortunately we ended up on some spambot's list faster than we hoped for. We have to finish up a couple other priorities first, but I hope to get this resolved within the next couple of days.

**Comment by Subtixx** [comment]
Maybe switching to a more versatile forum software like MyBB or SMF instead of PHPbb.

**Comment by Mellnik** [comment]
Or maybe vBulletin with Akismet anti spam and Captcha. Currently offers -$75 with coupon VB75OFF.

**Comment by OvermindDL1** [comment]
Discourse rather.

**Comment by Subtixx** [comment]
Doesn't matter what software... But PHPBB isn't the best choice is what I was trying to say.. PHPBB Forums always had terrible Anti-Spam mechanisms.. Everybody has his preferences, but PHPBB is by far the worst choice.

**Comment by Luke1410** [comment]
@Mellnik The concern with Akismet I'd have is privacy. While it's certainly a very useful product, it sends data to US servers. This is something I'd like to prevent, if possible. As things turn out, we'll likely require something which involves data integration of 3rd-parties (f.e. Google Captcha) to reliably reduce the amount of spam to a level which is manageable. Still, I'd like to keep this to the absolute minimum.

**Comment by dschwartz783** [comment]
Probably should just buy Xenforo. It’s pretty good.

**Comment by Luke1410** [comment]
We've been working over the past week to clean things up on the forum/webpage. We've upgraded the forum software, made adjustments to server side settings, and added additional anti-spam measurements. While cleaning up the existing spam posts/accounts is still wip, the BBS should be usable again already. I'll post some more details the following days about that in the forum (and will also get back to the posts which were previously hidden by other spam posts there). Feel free to bug me via PM (or re-post in the forum) if you were awaiting some urgent response.

Sorry for the inconvenience this would have caused.

**Comment by Subtixx** [comment]
Didn't really work...
https://www.slikesoft.com/forum/viewtopic.php?f=12&p=2361&sid=20e99c72862aa6c327f3419e5f1d5bf9#p2361
![https://i.imgur.com/NWMURid.png](https://i.imgur.com/NWMURid.png)

![https://i.imgur.com/jvNeyd4.png](https://i.imgur.com/jvNeyd4.png)

If you don't want to invest money, at least use MyBB. As I said earlier, PHPbb IS NOT up to the standards of 2017.

And then there are posts like this which make 0 sense (So probably bots too).
![https://i.imgur.com/eH471xv.png](https://i.imgur.com/eH471xv.png)

I don't know why you wouldn't listen to advice given to you.

**Comment by dschwartz783** [comment]
127,427 topics in General, seems to be all spam. I wonder what the record is...

Seriously, phpBB is really bad. I tried using it once. A self-corruption here, a poorly implemented feature there, and I ended up with Xenforo. There are some decent free options, but phpBB is not worth your time.

**Comment by Luke1410** [comment]
@Subtixx, dschwartz783: As stated above: cleaning up old spam is still WiP. However, it's now only limited to that one single forum (SLikeSoft General English). All other areas have already been cleaned up. Also note that the posts you see there are just getting bumped atm. There are no more new spam posts/threads.
We'll also have a couple further options at hand to go, if things get mezzy again in the future which we didn't pull just yet.

**Comment by Subtixx** [comment]
![https://camo.githubusercontent.com/79e2bb981ef1c5a96d58138f7e2504e7034cd820/68747470733a2f2f692e696d6775722e636f6d2f4e574d555269642e706e67](https://camo.githubusercontent.com/79e2bb981ef1c5a96d58138f7e2504e7034cd820/68747470733a2f2f692e696d6775722e636f6d2f4e574d555269642e706e67)
(Dec 11, 9:25 AM)
No new spam post uhu?

**Comment by Luke1410** [comment]
@Subtixx also thanks for pointing out the other spam posts which slipped through yesterday. Those have been removed as well now.
(FTR: the issue with bumping posts by spambots should have been resolved as well now)

**Comment by Luke1410** [comment]
All spam posts have been cleaned up now.
As announced earlier, here's the post with some further details on the spam activity, for those who are interested: https://www.slikesoft.com/forum/viewtopic.php?f=12&t=164164&p=170499#p170499
